Rakefile

```diff
@@ -74,8 +74,7 @@ namespace :style do
     desc "Run Chef Cookbook (Foodcritic) style checks"
     FoodCritic::Rake::LintTask.new(:foodcritic) do |t|
       t.options = {
-        fail_tags: ["any"],
-        tags: ["~FC007", "~FC011", "~FC031", "~FC045", "~supermarket"],
+         tags: ["~FC064", "~FC065", "~FC066", "~FC067"],
```

Comment:
There seems to be a bug in FC with Rake that is preventing ~supermarket from working here